webdriver/tests/element_click/bubbling.py

```diff
@@ -67,17 +68,19 @@ def test_spin_event_loop(session):
         <script>
         window.delayedClicks = [];
 
-        for (let level of document.querySelectorAll("div")) {
-          level.addEventListener("click", ({currentTarget}) => {
-            setTimeout(() => window.delayedClicks.push(currentTarget), 100);
+        var elements = document.querySelectorAll("div");
+        for (var level = 0; level < elements.length; level++) {
+          elements[level].addEventListener("click", function(clickEvent) {
+            var target = clickEvent.currentTarget;
+            setTimeout(function() { window.delayedClicks.push(target); }, 100);
           });
         }
         </script>
         """)
     three, two, one = session.find.css("div")
     one.click()
 
-    delayed_clicks = session.execute_script("return window.delayedClicks")
+    delayed_clicks = session.execute_script("return window.delayedClicks;")
```

Comment:
FWIW the semicolon shouldn’t be necessary as the injected script is wrapped in a function, and semicolons aren’t required in JS.

Reply:
Reverted to previous syntax, without semicolons.